proof/invariant-abstract/DetSchedInvs_AI.thy

```diff
@@ -3054,7 +3054,7 @@ lemma active_reply_scsE:
   using assms by (simp add: active_reply_scs_2_def active_if_reply_sc_at_2_def)
 
 definition valid_sched_2 where
-  "valid_sched_2 wk_vsa vbl ctime cdom ct it queues rlq sa etcbs tcb_sts tcb_scps tcb_faults sc_refill_cfgs sc_reps \<equiv>
+  "valid_sched_2 wk_vsa vbl riq ctime cdom ct it queues rlq sa etcbs tcb_sts tcb_scps tcb_faults sc_refill_cfgs sc_reps \<equiv>
```

Comment:
This is a neat solution and quite effective, but I'm slightly worried about the number of these boolean flags getting out of hand. It will be good to confirm with the rest of the team before merging this change, just to be safe.

Reply:
@mbrcknl in particular who might not be seeing all these PRs.

Reply:
I'm fine with this for now. This is another bundling problem where it would be nice to have a systematic way to patch or (de)select components of bundles. But that will take quite a lot of thought and disruption, so this is probably the best we can do right now.